itests/fevm_test.go

```diff
@@ -122,3 +143,30 @@ func TestFEVMETH0(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, *act.Address, eth0Addr)
 }
+
+// TestFEVMDelegateCall deploys two contracts and makes a delegate call transaction
+func TestFEVMDelegateCall(t *testing.T) {
+
+	ctx, cancel, client := setupFEVMTest(t)
+	defer cancel()
+
+	//install contract Actor
+	filenameActor := "contracts/DelegatecallActor.bin"
+	fromAddr, actorAddr := installContract(t, ctx, filenameActor, client)
+	//install contract Storage
+	filenameStorage := "contracts/DelegatecallStorage.bin"
+	fromAddrStorage, storageAddr := installContract(t, ctx, filenameStorage, client)
+	require.Equal(t, fromAddr, fromAddrStorage)
+
+	//call Contract Storage which makes a delegatecall to contract Actor
+
+	inputDataContract := inputDataFromFrom(t, ctx, client, actorAddr)
+	inputDataValue := inputDataFromArray([]byte{7})
+	inputData := append(inputDataContract, inputDataValue...)
+
+	result := invokeContract(t, ctx, client, fromAddr, storageAddr, "setVars(address,uint256)", inputData)
+	expectedResult, err := hex.DecodeString("0000000000000000000000000000000000000000000000000000000000000007")
```

Comment:
A comment specifying what this byte string means would be helpful

Reply:
ty! will add